test/Interop/Cxx/class/non-trivial-implicit-destructor-irgen.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s
```

Comment:
Please rename the test to start with `destructors-` (because it uses the C++ module with that name).